cmd/sling/tests/suite.db.template.tsv

```diff
@@ -3,18 +3,18 @@ n	test_name	source_conn	source_stream	source_options	stream_config	target_conn	t
 2	csv_full_refresh_delimiter	local	file://tests/files/test5.csv	{}		[conn]	[schema].[table]_2	full-refresh			"{""post_sql"" : ""{drop_view}""}"	"{""validation_row_count"": "">0""}"
 3	discover_table					[conn]	[schema].[table]	discover				"{""validation_contains"": ""create_dt"", ""validation_row_count"": ""11"", ""level"": ""column""}"
 4	parquet_snapshot	local	file://tests/files/test1.parquet	{}		[conn]	[schema].[table]_snapshot	snapshot			"{""post_sql"" : ""drop table [schema].[table]_2""}"	"{""validation_row_count"": "">999"", ""validation_types"": {""code"":""decimal"",""create_dt"":""timestamp"",""date"":""timestamp"",""rating"":""decimal"",""target"":""bool""}}"
-5	parquet_truncate	local	file://tests/files/test1.parquet	"{""columns"": {""rating"": ""float""}}"		[conn]	[schema].[table]_truncate	truncate			"{""pre_sql"": ""drop table [schema].[table]_snapshot "", ""post_sql"" : ""drop table [schema].[table]_truncate""}"	
+5	parquet_truncate	local	file://tests/files/test1.parquet	"{""columns"": {""rating"": ""float""}}"		[conn]	[schema].[table]_truncate	truncate			"{""pre_sql"": ""drop table [schema].[table]_snapshot "", ""post_sql"" : ""drop table [schema].[table]_truncate""}"
```

Comment:
![image](https://github.com/user-attachments/assets/8b6baca9-0eaa-4217-8ec4-01b5e0d71ee7)
![image](https://github.com/user-attachments/assets/38a6275b-5916-459c-a4a3-63d7f1eec68e)

last time I struggled with this display as well, is there any reason we change the blank?
https://github.com/slingdata-io/sling-cli/blob/5e59b15b6c92d68f2b091118951aa6bafd7be6f6/cmd/sling/tests/suite.db.template.tsv

https://github.com/slingdata-io/sling-cli/blob/1.2.21/cmd/sling/tests/suite.db.template.tsv

Reply:
Ah my editor did some autoformat; thanks!